### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by anniethewebcoder**
week 2 homework

### CODE REVIEW

File: 01-node-tutorial/answers/16-streams.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { createReadStream } = require('fs')
+
+//const stream = createReadStream('temp.txt', { encoding: 'utf8' })
+const stream = createReadStream('temp.txt', { highWaterMark: 64})
+
+stream.on('data', (result) => {
+  console.log(result)
+})
+stream.on('error', (err) => console.log(err))
```
Comment (alexgs) [inline 01-node-tutorial/answers/16-streams.js:9]: You're missing a counter to keep track of how many chunks are loaded.

File: 01-node-tutorial/answers/writeWithPromisesThen.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const { writeFile, readFile } = require('fs').promises
+
+const line_one = 'Alex loves Complete Breakfast'
+const line_two = 'Alex also loves Salmon Dinner'
+const line_three = 'You\'ll know if he loves it when he say, "Hey, Awesome! I love this stuff!"'
+
+writeFile('temp.txt', `${line_one}\n${line_two}\n${line_three}`)
+    .then((res) => console.log(res))
```
Comment (alexgs) [inline 01-node-tutorial/answers/writeWithPromisesThen.js:8]: In this part of the assignment, you're supposed to chain several `.then` callbacks. You should write one line with each callback, for three lines total.

**Review by alexgs — APPROVED** [review]
You didn't implement `writeWithPromisesThen.js` correctly, but I think you got the gist of it on this assignment.

File: 01-node-tutorial/answers/16-streams.js (hunk 1 above)
Comment (anniethewebcoder) [reply 01-node-tutorial/answers/16-streams.js:8]: I wrote in the counter.

File: 01-node-tutorial/answers/writeWithPromisesThen.js (hunk 2 above)
Comment (anniethewebcoder) [reply 01-node-tutorial/answers/writeWithPromisesThen.js]: I made one more "then" chain to write one line at a time.

**Pull request merged**